src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import { Person } from '../../types';
+import cn from 'classnames';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
```

Comment:
It's not the link - it's row or item in the table

Reply:
Fixed.